### DISCUSSION TITLE
Enhancement: Multi-Tenant Auth

### DISCUSSION THREAD

**Discussion by Dannypeja** [danny-avila/LibreChat#4431] [Feature Requests & Suggestions]
### What features would you like to see added?

Use Multiple SSO Tenants for Login. Right now, it is only possible to use a single one of a sort: OpenID or Discord. 
Would to being able to use multiple OpenID configs. 

### More details

nothing more :=) 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is already implemented. You can use OpenID and Discord, but not multiple OpenID configurations.

https://www.librechat.ai/docs/configuration/authentication/OAuth2-OIDC/discord#env-configuration
